internal/cmd/controller/agentmanagement/controllers/resources/data.go

```diff
@@ -34,6 +34,11 @@ func ApplyBootstrapResources(systemNamespace, systemRegistrationNamespace string
 					APIGroups: []string{fleet.SchemeGroupVersion.Group},
 					Resources: []string{fleet.BundleDeploymentResourceNamePlural + "/status"},
 				},
+				{
```

Comment:
We grant this permission, even if ociutils.ExperimentalOCIIsEnabled is false?

Reply:
good point.
This approach is passing the env variable to `fleet-controller` and `gitjob` (upstream) only.
Do we have a mechanism to pass env variables to the agent? 
How can we be sure that the env variable is passed when registration is initiated by the `agent`? 
We could have the case of an upstream supporting OCI and downstream that doesn't... which means the `bundledeployments` it accesses may not be valid... 
Good topic for discussion. :)

Reply:
It might work if we put it in the agents config.